lib/commands/std-authenticate.ts

```diff
@@ -0,0 +1,23 @@
+/* Copyright (c) 2024. SailPoint Technologies, Inc. All rights reserved. */
+
+import { AccountSchema, Attributes, ObjectInput, ObjectOutput, Permission } from './command'
+
+/**
+ * Input object of `std:authenticate` command
+ */
+export type StdAuthenticateInput = ObjectInput & {
+    username: string
+	password: string
+    options? : any
+	schema: AccountSchema
```

Comment:
Is `schema` always required?

Reply:
Schema will be needed, since after authentication, RO of the identity is supposed to be returned back from the connector.